src/ExtensionsSample/Samples/DocumentDBSamples.cs

```diff
@@ -52,6 +53,19 @@ public static class DocumentDBSamples
             item["text"] = "Text changed!";
         }
 
+        // Document query input binding
+        //   This binding requires teh 'SqlQuery' property to be specified. The binding uses
+        //   that string to issue a query against the specified collection. The resulting collection is passed to the 
+        //   function parameter.
+        //
+        // This example uses the binding template "{QueueTrigger}" to specify that the value of 'c.isCompleted' should come
+        // from the string value of the queued item.
+        public static void QueryDocument(
+            [QueueTrigger("samples-documentdb-csharp")] string input,
+            [DocumentDB("ItemDb", "ItemCollection", SqlQuery = "SELECT c.id, c.text FROM c where c.isCompleted = {QueueTrigger}")] IEnumerable<ItemDoc> items)
```

Comment:
Good point. They do support SqlParameters. I'll have to think through how we make that work.